### PR TITLE
Lpds 28 label component

### PR DESCRIPTION
**Pull request by naufal22hadi**
# Description, Motivation and Context

`LPLabel` is a label pattern component that can be changed based on the style. There are 3 style of `LPLabel` :

1. Basic (Default)
2. Product
3. Membership

# JIRA Ticket

[LPDS-28](https://lionparcel.atlassian.net/browse/LPDS-28)

# What is the destination branch of this PR?

Master

| Description | Image |
| -- | -- |
| Basic | ezgif com-gif-maker (3) |
| Product | ezgif com-gif-maker (4) |
| Membership | label_membership |

### CODE REVIEW

**Review by falih-mulyana — APPROVED** [review]
LGTM

File: core/src/main/java/com/lionparcel/commonandroid/label/res/drawable/bg_selector_label_basic.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
```
Comment (daniimulyana) [inline core/src/main/java/com/lionparcel/commonandroid/label/res/drawable/bg_selector_label_basic.xml:2]: ini fungsi `layer-list` buat apa mas?

**Review by daniimulyana — APPROVED** [review]

File: core/src/main/java/com/lionparcel/commonandroid/label/res/drawable/bg_selector_label_basic.xml (hunk 1 above)
Comment (naufal22hadi) [reply core/src/main/java/com/lionparcel/commonandroid/label/res/drawable/bg_selector_label_basic.xml:2]: lupa gw ganti mas, harusnya make selector

Comment (naufal22hadi) [reply core/src/main/java/com/lionparcel/commonandroid/label/res/drawable/bg_selector_label_basic.xml:2]: sebenernya shape aja cukup sih, wait gw ganti mas

**Review by riomanik — APPROVED** [review]

**Review by fadhilahsaputro — APPROVED** [review]

**Review by fajardarwis — APPROVED** [review]

**Pull request merged**